services/motion/motion.go

```diff
@@ -96,16 +96,16 @@ type ListPlanStatusesReq struct {
 	Extra           map[string]interface{}
 }
 
-// Plan represents a motion plan.
-type Plan struct {
+// PlanWithMetadata represents a motion plan with additional metadata used by the motion service.
+type PlanWithMetadata struct {
```

Comment:
This is a breaking change for the Go SDK. Do we care about that?

Reply:
It's not the only one I think.  I will call it out in the breaking changes channel before merging